`maxRetries` field missing from ClientConfig type

The docs say I can specify `maxRetries` when instantiating a client, but typescript does not allow it.

#### Environment details

  - OS: macOS 10.15.3
  - Node.js version: 10.17.0
  - npm version: 6.11.3
  - `@google-cloud/pubsub` version: 1.6.0
  - typescript version: 3.7.2

#### Steps to reproduce

  1. create a new file containing `new Pubsub({projectId: "whatever", maxRetries: 5})`
  2. run `tsc --noEmit`

```
error TS2345: Argument of type '{ projectId: string; maxRetries: number; }' is not assignable to parameter of type 'ClientConfig'.
  Object literal may only specify known properties, and 'maxRetries' does not exist in type 'ClientConfig'.
```

--- a/src/pubsub.ts
+++ b/src/pubsub.ts
@@ -203,8 +203,6 @@ interface GetClientCallback {
  * @property {boolean} [autoRetry=true] Automatically retry requests if the
  *     response is related to rate limits or certain intermittent server errors.
  *     We will exponentially backoff subsequent requests by default.
- * @property {number} [maxRetries=3] Maximum number of automatic retries
- *     attempted before returning the error.
  * @property {Constructor} [promise] Custom promise module to use instead of
  *     native Promises.
  */

